frontend/.prettierrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "singleQuote": true,
+  "trailingComma": "es5",
+  "tabWidth": 2,
+  "semi": true,
```

Comment:
The `"tabWidth": 2` and  `"semi": true,` are the defaults, so could remove these.